content/html/concepts/tags/terms/noscript/noscript.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '<noscript>'
+Description: 'Displays if the user disabled Javascript in the browser.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<noscript>` tag will display its contents when the user disabled Javascript for that site.
+  
+## Syntax
+  
+```html
+<noscript>
+  <!-- What should display if JS is not enabled -->
+</noscript>
+```
+
+The tag has no attributes and what is nested within the tag will be displayed (if JS is not enabled).
+  
+## Example
+
+```html
+<script href="someScript.js"></script>
```

Comment:
What is the purpose of this `<script>` tag being here?

Reply:
@Dusch4593 Why would you use the no script tag if you don’t have JavaScript?

Reply:
I'm saying that if the `<script>` tag plays a role in this example, it should be explained in text-form above the snippet (but below the "Example" heading).

Also, `href` is not a valid attribute with the `<script>` tag; use `src="someScript.js"`, instead.